src/services/userManager.js

```diff
@@ -0,0 +1,15 @@
+import {createUserManager} from 'redux-oidc';
+
+const userManagerConfig = {
+  client_id: 'openidconnectcert',
+  redirect_uri: `${window.location.protocol}//${window.location.host}/callback`,
```

Comment:
yeah `protocol` => `http:` and the `host` is equivalent to `hostname + : + port`